common/lib/xmodule/xmodule/js/src/video/10_main.js

```diff
@@ -138,6 +140,10 @@
                     initialize(state, element);
                 }
 
+                if (autoAdvanceEnabled) {
+                    mainVideoModules.push(VideoAutoAdvanceControl);
+                }
```

Comment:
Bugs can happen in any case depending on the external code. E.g. there could be other part of edx that expected the VideoAutoModule to be always loaded, and even code that called some of the functions, and they would fail if not present (I bet that's the case with some of the other >10 modules from the list). But if this happened it would need to be fixed. And you told me it doesn't happen, so it's fine like this.